odoo/models.py

```diff
@@ -76,6 +76,9 @@
 
 AUTOINIT_RECALCULATE_STORED_FIELDS = 1000
 
+INSERT_BATCH_SIZE = 100
```

Comment:
Just an idea: what about moving this constant as a class attribute ?
This would enable to specify different batch sizes on different models...

Reply:
Mhhh, it looks like too much fine-tuning. Should we do the same for `AUTOINIT_RECALCULATE_STORED_FIELDS`, `IN_MAX`, `PREFETCH_MAX` ? 
In the future, if we really need this, we will do. But for now, it sounds like adding noise.

Reply:
Ok, good enough, then.